styles.css

```diff
@@ -0,0 +1,116 @@
+body {
+  background-color: #1c1a19;
+  margin: 0;
+}
+
+.first_section_div {
+  height: auto;
+  width: 347px;
+}
```

Comment:
- [ ] Kindly make sure to adjust your `width`on this container in order to meet the provided design. :width:

    - ### Your design
 
     ![image](https://user-images.githubusercontent.com/87197412/193079137-6bc3d49e-69a7-4964-9c22-9e3c686a3db4.png)

    - ### Figma design
    ![image](https://user-images.githubusercontent.com/87197412/193079341-d3d38c17-d09c-4c18-bd38-0c25b19f1bac.png)